test/integration-tests/tests/irrelevant-entities-tests.test.ts

```diff
@@ -0,0 +1,58 @@
+import { graphqlRawRequest } from '../test-server/client';
+
+const headers = { 'reality-id': 1, 'data-version': 1 };
+
+describe('irrelevant entities tests', () => {
+    test('2 irrelevant entities not starting with f', async () => {
+        const queryBook = `{
+          booksStartsWith(startsWith:"f"){
+            id,
+            title
+          }
+        }`;
+        const { extensions }: any = await graphqlRawRequest(queryBook, headers, {});
+
+        expect(extensions.irrelevantEntities.booksStartsWith.length).toBe(2);
```

Comment:
also check the entities that are returned equal to the expected entities

Reply:
in all tests

Reply:
but they are generated in runtime (guid) i cant expect them

Reply:
you can check other properties